t04_refactored.py

```diff
@@ -177,6 +177,28 @@ def team_13_adv():
 def team_14_adv():
     pass
     # TODO Add your code here
+    direction = input("are you going [North/East/South/West]?")
+    if direction == "North":
+        print("You didn't die. In fact, you made it out of the cave and back into society. Yay.")
+        sleep(delay)
+    elif direction == "South":
+        print("You unfortunately fell off a cliff and are about to die.")
+        dead == True
```

Comment:
This is not setting dead to true, it is checking if dead is equal to true:

`dead = True`